DEVDOC.md

```diff
@@ -137,7 +137,13 @@ Follow these steps to use the delve debugger for Go with VSCode to debug `gapis`
 
 2. Create a `launch.json` file under the workspace directory with `Ctrl + Shift + P` and `Debug: Open launch.json`
 
-3. Paste the following as one of the launch configurations. This will ensure that there is a launch configuration for attaching to Delve.
+3. There will be two settings file, one is for global the other one is local to the project.
```

Comment:
typo: ...two settings file**s**

I would recommend the present tense: `There are to settings files...`